screens/datacontact.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
We should store dummy data under the data folder and create a data class respectively. Please check data/dummy-data.js for more information.